lib/msg.rb

```diff
@@ -0,0 +1,34 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Msg
+  attr_reader :delivery_time, :credentials
+
+  def initialize
+    @credentials = Dotenv.load
+  end
+
+  def expected_time
+    @delivery_time = (Time.new + 3600).hour.to_s + ":" + (Time.new + 3600).min.to_s
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)